service/service.go

```diff
@@ -100,25 +100,25 @@ func init() {
 		}
 		os.Exit(0)
 	}
+	//todo: refactor this to LISTEN_ADDRESS
+	Config().RegisterOptionalParameter("LISTEN_HOST", "0.0.0.0", "service.listen.host", "The address the service listens on")
+	Config().RegisterOptionalParameter("LISTEN_PORT", "8080", "service.listen.port", "The port the service listens on")
+	Config().Parse()
 }
 
 // New returns a new Service object
 func New(name string) *Service {
 
-	if _, present := os.LookupEnv("LISTEN_HOST"); present {
-		listenHost = os.Getenv("LISTEN_HOST")
+	// check if name has at least one character
+	if len(name) < 1 {
```

Comment:
Why not `len(name) == 0` or `name == ""` ?
I find `len(name) < 1` a bit confusing in the first place.

Reply:
I believe you can do it in a bazillion ways, this is mine, I even left a inline comment ;)

Reply:
okay.